File: code/stable/projectile/src/csharp/InputFormat.cs
/** \file InputFormat.cs
    \author Samuel J. Crawford, Brooks MacLachlan, and W. Spencer Smith
    \brief Provides the function for reading inputs
*/
using System;
using System.IO;
using System.Collections;
using System.Collections.Generic;

public class InputFormat {
    
    /** \brief Reads input from a file with the given file name
        \param inParams structure holding the input values
        \param filename name of the input file
    */
    public static void get_input(InputParameters inParams, string filename) {
        StreamReader infile;
        infile = new StreamReader(filename);
        infile.ReadLine();
        inParams.v_launch = Double.Parse(infile.ReadLine());
        infile.ReadLine();
        inParams.theta = Double.Parse(infile.ReadLine());
        infile.ReadLine();
        inParams.p_target = Double.Parse(infile.ReadLine());
        infile.Close();
    }
}

File: code/stable/projectile/src/python/InputFormat.py
## \file InputFormat.py
# \author Samuel J. Crawford, Brooks MacLachlan, and W. Spencer Smith
# \brief Provides the function for reading inputs
from __future__ import print_function
import sys
import math

import InputParameters

## \brief Reads input from a file with the given file name
# \param inParams structure holding the input values
# \param filename name of the input file
def get_input(inParams, filename):
    infile = open(filename, "r")
    infile.readline()
    inParams.v_launch = float(infile.readline())
    infile.readline()
    inParams.theta = float(infile.readline())
    infile.readline()
    inParams.p_target = float(infile.readline())
    infile.close()
